java/code/src/com/redhat/rhn/frontend/strings/java/StringResource_en_US.xml

```diff
@@ -1921,19 +1921,19 @@
 &lt;p&gt;To resolve this issue, please check the following things:&lt;/p&gt;
 &lt;ul&gt;
 &lt;li&gt;
-Is the 'rhn-tools' child software channel for this kickstart profile's base
+Is the tools child software channel for this kickstart profile's base
```

Comment:
It sound not good.

Should we just call it "Client Tools Channel" and remove the "software"? Also in the text below?

Reply:
That sounds good.